patternfly/patternfly-elements
# CI: Reduce storybook build output

Currently the storybook build is outputting every % step as it builds to the CI log.  This makes the log hard to parse.  Reducing this by adding the `--quiet` and adding the `--loglevel error`.

### Testing instructions

Check the [output from the build](https://github.com/patternfly/patternfly-elements/runs/1856482240?check_suite_focus=true#step:7:1327)
- Click to expand "Build"
- Scroll to the bottom, check that the storybook build only reports a failure.

Commands affect:
- `npm run storybook`
- `npm run build-storybook`

### Ready-for-merge Checklist

Check off items as they are completed.  Feel free to delete items if they are not applicable.


### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
